lib/config.js

```diff
@@ -1,14 +1,16 @@
 /*
- * Copyright (c) 2012-2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2021 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
 const config = require('bedrock').config;
 const fs = require('fs');
 const path = require('path');
 
+const VueLoaderPlugin = require('vue-loader/lib/plugin');
+
 const DEFAULT_FRAMEWORKS = ['mocha', 'chai'];
-const DEFAULT_PREPROCESSORS = ['babel', 'webpack'];
+const DEFAULT_PREPROCESSORS = ['babel', 'webpack', 'coverage'];
```

Comment:
we had trouble getting karma's `karma-coverage` to work with webpack (it reported back incorrect lines and had the wrong percentages). What turned out to be easier was using these 2 libs:

```js
    "@jsdevtools/coverage-istanbul-loader": "^3.0.5",
```json
    "karma-coverage-istanbul-reporter": "^3.0.3"
```

then:

```js
config.karma.config.reporters = ['mocha', 'coverage-istanbul'];
config.karma.config.coverageIstanbulReporter = {
  reports: ['html', 'text'],
  dir: path.join(__dirname, 'coverage'),
};
```